pkg/rewards/claim.go

```diff
@@ -0,0 +1,312 @@
+package rewards
+
+import (
+	"errors"
+	"fmt"
+	"log/slog"
+	"math/big"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/common"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/common/flags"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/types"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/utils"
+
+	contractrewardscoordinator "github.com/Layr-Labs/eigenlayer-contracts/pkg/bindings/IRewardsCoordinator"
+
+	"github.com/Layr-Labs/eigenlayer-rewards-proofs/pkg/claimgen"
+	"github.com/Layr-Labs/eigenlayer-rewards-proofs/pkg/proofDataFetcher/httpProofDataFetcher"
+
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts"
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
+	"github.com/Layr-Labs/eigensdk-go/chainio/txmgr"
+	rewardscoordinator "github.com/Layr-Labs/eigensdk-go/contracts/bindings/IRewardsCoordinator"
```

Comment:
This and line 18 _should_ be the same, so not sure why you need both. Line 18 is the correct one to use

Reply:
basically what's happening here is - eigenlayer-cli uses `eigensdk` bindings (which we use heavily in CLI for reading/writing onchain) and rewards proof is using eigenlayer-contracts bindings. I have talked to @samlaf about using same eventually but till then we need to import to have conversion between contracts bindings and eigensdk bindings